demo/kubernetes/rook.yml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: rest
+  labels:
+    app: rook
+    role: rest
+spec:
+  type: NodePort
+  externalIPs: [ "172.17.4.101" ]
+  ports:
+  - port: 8124
```

Comment:
It is called out in the readme as something that needs to be changed if you are in a different environment. It works if the user uses the coreos-kubernetes repo to launch k8s via multi-node vagrant which was the demo target.